### ISSUE TITLE
Desktop Comet Lake SMBIOS

### ISSUE THREAD

**Issue by ctokun** [dortania/bugtracker#90]
The [Desktop Comet Lake guide](https://dortania.github.io/OpenCore-Install-Guide/config.plist/comet-lake.html#platforminfo) recommends using SMBIOS iMac20,1 for i7-10700K and lower and iMac20,2 for i9-10850K and higher. Choosing iMac20,1 for my i7-10700K caused GPU errors that were not resolvable by any methods in the troubleshooting guide. The only fix was changing the SMBIOS to iMac20,2. It may be helpful to note that iMac20,2 may be necessary for i7-10700K.

**Comment by khronokernel** [comment]
Hmm interesting, first I've heard of it however not too many reports with Comet Lake in general. What motherboard, display output and AAPL,ig-platform-id were you using?

**Comment by ctokun** [comment]
Motherboard is Z490 Gigabyte Vision G and I tried both HDMI and DP for display output. The AAPL,ig-platform-id was 07009B3E. I just read the [macOS update changes](https://dortania.github.io/OpenCore-Post-Install/universal/update.html#updating-macos) and saw that the UHD 630's framebuffer broke in 10.15.5 and needed to be changed to 00009B3E. I just tried going back to using iMac20,1 SMBIOS but used 00009B3E for the AAPL,ig-platform-id instead and it worked fine. No clue why using iMac20,2 bypasses the framebuffer issue.
